Classes/Job/JobManager.php

```diff
@@ -175,6 +176,18 @@ public function peek(string $queueName, int $limit = 1): array
         }, $messages);
     }
 
+    /**
+     * This method is here to be called by queues if they reached local polling timeouts, if this applies
+     *
+     * @throws InterruptException
+     */
+    public function interruptMe(): void
```

Comment:
nitpick: I find the name `interruptMe` a little weird. Why not just `JobManager::interrupt()`

Reply:
Naming things is always the hardes task in programming. The idea had been, to find a name, which makes clear for the using classes, that this is more a less a signal to the job manager, that this would be a safe point for an interruption. Your suggestion is totally fine for me, if this makes more sense for you.

Reply:
I was confused by the imperative verb. But since this doesn't actually interrupt anything I would suggest to call this something like `processPcntlSignals()` or so instead.. But this leaves me wondering: Why do we need this methond in the `JobManager` at all? Can't we just move this line to the CommandController?

Reply:
It is there to make it easy for queues to signal their "safe state" to the job manager. They should not need to be aware of the CommandController. And if no Signal handler was registered, it just do nothing, so means no harm to call.

If you think the signal handler could be managed by the JobManager I'm with you. I had reasons to nut put it there - but I currently do not remember